r/R/dplyr-functions.R

```diff
@@ -672,6 +672,10 @@ nse_funcs$strptime <- function(x, format = "%Y-%m-%d %H:%M:%S", tz = NULL, unit
   Expression$create("strptime", x, options = list(format = format, unit = unit))
 }
 
+nse_funcs$strftime <- function(x, format = "%Y-%m-%d %H:%M:%S", locale="C") {
```

Comment:
In the R bindings for these functions, the arguments and values should match the base R or tidyverse function that is being implemented.  This doesn't quite match - here are the docs for base R's `strftime()`: https://stat.ethz.ch/R-manual/R-devel/library/base/html/strptime.html

Reply:
Done, please check if I got it right. Note that `strftime` kernel doesn't support `tz` parameter at the moment.